app/resources/brave_generated_resources_bg.xtb

```diff
@@ -351,7 +354,23 @@
         Можете да прочетете <ph name="BEGIN_LINK_BUILD_INSTRUCTIONS"/>инструкции как да изтеглите и изградите за себе си<ph name="END_LINK_BUILD_INSTRUCTIONS"/> конкретния <ph name="BEGIN_LINK_SOURCE_CODE"/>изходен код, използван за създаването на това копие<ph name="END_LINK_SOURCE_CODE"/>.</translation>
 <translation id="4474155171896946103">Запазване на отметки към всички раздели…</translation>
 <translation id="2336228925368920074">Запазване на отметки към всички раздели...</translation>
+<translation id="4235920711083356755">Показване на бутона VPN</translation>
```

Comment:
```suggestion
<translation id="4235920711083356755">Показване на VPN бутона</translation>
```

It makes more sense to say "Show the VPN button" rather than "Show the button VPN".